src/twisted/newsfragments/12134.feature

```diff
@@ -0,0 +1 @@
+Added support for `cipher` in `serverFromString` function
```

Comment:
The current test read more like a commit mesage.

The newsframgment, is for the public release notes.

It's important to clearly explain which API is targeted by this change, and what is this change all about.

The project convention for the release notes, it to start the message with the canonical name of the API that was changed

```suggestion
twisted.internet.endpoint.serverFromString now supports defining a cipher list. 
```

Reply:
[#f991e5a](https://github.com/twisted/twisted/pull/12137/commits/f991e5a4aa8b557cb19c222f876940b12cc0c0b6)